qa/tasks/cephfs/test_scrub.py

```diff
@@ -144,6 +148,35 @@ def _scrub(self, workload, workers=1):
                 errors[0].exception, errors[0].backtrace
             ))
 
+    def _scrub_new_files(self, workload):
+        """
+        That scrubbing new files does not lead to errors
+        """
+        workload.create_files(1000)
+
+        out_json = self.fs.rank_tell(["scrub", "start", "/", "recursive"])
+        while True:
+            out_json = self.fs.rank_tell(["scrub", "status"])
+            if out_json['status'] == "no active scrubs running":
+                break;
+            time.sleep(10)
```

Comment:
It's good you're checking here for the asynchronous scrub to complete but I've noticed none of the other tests do this. Did you hit a race condition in testing?

I think it would be valuable to have a "synchronous" scrub method in `cephfs_test_case.py` which waits for a scrub to complete. How about moving it there? Also please take a look at `teuthology.contextutil.safe_while` for handling the retries/timeout instead.

Reply:
> It's good you're checking here for the asynchronous scrub to complete but I've noticed none of the other tests do this. Did you hit a race condition in testing?

I never tested for race conditions. I always thought waiting for the asynchronous scrub to complete was the right thing to do.

Reply:
> I think it would be valuable to have a "synchronous" scrub method in `cephfs_test_case.py` which waits for a scrub to complete. How about moving it there? Also please take a look at `teuthology.contextutil.safe_while` for handling the retries/timeout instead.

done